### DISCUSSION TITLE
How to implement stale-while-revalidate

### DISCUSSION THREAD

**Discussion by j-waaang** [arthurfiorette/axios-cache-interceptor#866] [Q&A]
Does axios-cache-interceptor support such a functionality? I saw "cache-first" mentioned on one of the documentation pages, but there was no information about it.
If a cache entry is stale I want it still be served and I want to cache entry to be refreshed so the next request will receive fresh data.

**Comment by arthurfiorette** [comment]
https://axios-cache-interceptor.js.org/config/request-specifics#cache-hydrate
